tests/page-objects/tabElementsParametisado.ts

```diff
@@ -0,0 +1,46 @@
+import {Page, Locator, expect} from '@playwright/test';
```

Comment:
I might want to use just english naming of files if you are using english in all other places.

Reply:
same with comments.

Reply:
to that note - if you dont need the comments, delete them before you commit them.